docs/content/reference/resource-schema/core-schema/application-schema/index.md

```diff
@@ -3,7 +3,7 @@ type: docs
 title: "Application reference"
 linkTitle: "Application"
 description: "Learn how to define an application"
-weight: 100
+weight: 200
```

Comment:
this change is to ensure that the ordering of links in the sidebar becomes: Environment, Application, Container